app/gui/src/presenter/graph/state.rs

```diff
@@ -659,18 +659,24 @@ impl<'a> ViewChange<'a> {
     }
 
     /// Set the node expression.
-    pub fn set_node_expression(&self, id: ViewNodeId, expression: String) -> Option<AstNodeId> {
+    pub fn set_node_expression(&self, id: ViewNodeId, expression: String) -> Option<()> {
```

Comment:
We could just return bool, I guess? And what exactly information is returned now? Before we returned the AST ID of the node whose expression needs to be updated. Why we return `()`? It looks like there is no need to update AST in controllers anymore, what sounds strange to me.

Reply:
The return value is not used anywhere. So this function can return nothing (if we remove usages of the `?` operator).

Reply:
Hmm, I would still stay with the old behavior. It's a public method, after all, and all methods of the `ViewChange` returns some information about what needs to be updated in controllers. The fact we ignore it right now seems to be just an incidental thing.